x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/provider.tsx

```diff
@@ -32,7 +37,8 @@ export const Provider = React.memo<OwnProps>(({ dataProvider }) => {
       toggleExcludedProvider={noop}
       toggleEnabledProvider={noop}
       toggleTypeProvider={noop}
-      val={dataProvider.queryMatch.displayValue || dataProvider.queryMatch.value}
+      displayValue={String(dataProvider.queryMatch.displayValue || dataProvider.queryMatch.value)}
```

Comment:
This change looks good

Reply:
nit: consider using the nullish coalescing operator (`??`) here

Reply:
changed